### ISSUE TITLE
DiscoverDeviceAsync method not found in version 2.1.1

### ISSUE THREAD

**Issue by marcoseraphin** [xabre/xamarin-bluetooth-le#414]
I am using version 2.1.1, but I could not find the DiscoverDeviceAsync() method in the Adapter (Interface) 

var device = adapter.DiscoverDeviceAsync(dev => dev.Name.Equals("Adrian")));

like described in the blog:

https://smstuebe.de/2016/05/13/blev1.0/

Thanx

Marco

**Comment by smsissuechecker** [comment]
Hi @marcoseraphin,

I'm the friendly issue checker.
It seems like (100.00 %) you haven't used our issue template :cry: I think it is very frustrating for the repository owners, if you ignore them.

If you think it's fine to make an exception, just ignore this message.
**But if you think it was a mistake to delete the template, please close the issue and create a new one.**

Thanks!

**Comment by MKuckert** [comment]
But it's there:
https://github.com/xabre/xamarin-bluetooth-le/blob/master/Source/Plugin.BLE.Abstractions/Extensions/AdapterExtenstion.cs#L55

You may have to add a using to `Plugin.BLE.Abstractions.Extensions`.

**Comment by marcoseraphin** [comment]
Yes, thank you....don't know why I did not found it in February...
